packages/adyen-salesforce-pwa/lib/api/middleware/webhook.js

```diff
@@ -92,8 +72,9 @@ function parseNotification(req, res, next) {
         Logger.info('AdyenNotification', JSON.stringify(res.locals.notification))
         return next()
     } catch (err) {
+        Logger.error('parseNotification', err.stack)
         return next(err)
```

Comment:
If this error message is also reflected to Adyen while accepting the webhook through the endpoint, the merchant's webhook queue might be blocked due to failing webhooks. Any failing webhook will be put on the retry queue by Adyen. However, the result will always be same if there is a parsing issue. Therefore, the webhook will be stuck at failed state.